article/Method/alpha-k-layer-combination.tex

```diff
@@ -0,0 +1,33 @@
+\subsection{Optimizing layer combination}
+\subsubsection{Removing $\alpha_k$}
+% Overvej om denne sektion bør flyttes til appendix med en reference til den.
+Before trying to optimize $\alpha_k$ we wanted to investigate what impact it has.
+The simplest experiment is to remove it.
+If this results in small changes in performance, it can indicate that it would not be worthwhile to pursue the opportunity to optimize $\alpha_k$.
```

Comment:
```suggestion
If this results in small changes in performance, it could indicate that it would not be worthwhile to pursue the opportunity to optimize $\alpha_k$.
```